lib/ui/hooks.dart

```diff
@@ -41,7 +41,11 @@ void _updateWindowMetrics(double devicePixelRatio,
 
 typedef String LocaleClosure();
 
-String _localeClosure() => window._locale.toString();
+String _localeClosure() {
+  const String enUS = 'en_US';
+  final String locale = window?._locale?.toString() ?? enUS;
+  return (locale == '_') ? enUS : locale;
```

Comment:
Why is it `_` in the first place versus just being null or the empty string?  Are the tests that failed valid in wanting it to start out as `_`?

Reply:
The Locale is initialized from the engine with empty strings for the language and the country codes. Some logic in the tests and/or the framework appears to rely on that behavior. Whether that makes sense or not, I wouldn't venture to guess.